Taskfile.yml

```diff
@@ -29,8 +29,13 @@ includes:
     dir: website
 
 tasks:
+  # Set up a local environment
   postpull:
-    deps: ['workspaces:clean', 'workspaces:build']
+    cmds: 
+      - task: 'workspaces:clean'
+      - FLAGS_FILE=.env.local task workspaces:setenv
+      - task: 'workspaces:build'
```

Comment:
Is there a better way to specify env variable setup using task?

Reply:
this is fine imo, but there are other syntaxes for env stuff, https://taskfile.dev/#/usage?id=environment-variables

Reply:
Overriding the local flags file with this new `.env.local` has just cost me a whole good of head scratching. A developers local `.env` should remain untouched at all times, it's how developer A can run different local config than developer B (as they are working on different things).

Reply:
Reverted as part of #1077.